### DISCUSSION TITLE
假面愚者招聘信息

### DISCUSSION THREAD

**Discussion by siler-wolf** [share121/inter-knot#1750] [其他]
<img width="500" alt="cc7a1029ef7377814228687f0a68d872" src="https://github.com/user-attachments/assets/cda227a8-6328-4184-9d25-d3aedf5c5bce">
##嘿！是你！就是你！
你是否因为生活太过无趣而感到迷茫？
你是否因为生活太过枯燥而焦虑不安？
这时，你就需要乐子来帮忙啦！
加入我们吧，
你将会获得最有趣的笑话；
最新奇的故事；
以及... 最荒诞的乐子。

「酒馆」静候您的到来。
and:
下方链接是酒馆官网
https://www.bilibili.com/video/BV1GJ411x7h7?vd_source=29e47e6ea2278554e0b0515b27adfb3f

**Comment by siler-wolf** [comment]
人呢？

**Comment by lllliufeng** [comment]
欢迎来到绳网，官方网址：
[https://www.绳网.top](https://www.bilibili.com/video/BV1GJ411x7h7/)

**Comment by H-7942** [comment]
愿者上钩

**Comment by inkluv** [comment]
名字是故意缺字母的还是不小心的（doge

**Comment by FlameF6** [comment]
看得出来是真假面愚者

**Comment by lamtt26** [comment]
6，被骗了那么多次，还是没有认出来，裂开

**Comment by 233box** [comment]
你 被 骗 了

**Comment by su2569** [comment]
6

**Comment by paowaer566** [comment]
年年考，年年错

**Comment by 54188-sys** [comment]
可恶，居然着到了。

**Comment by TheFool-rgb** [comment]
哈，还真有人，真有乐子（
